### PR TITLE
feat: add oslo endpoint

### PR DESCRIPTION
**Pull request by alainthoen**

### CODE REVIEW

File: src/PostalRegistry.Api.Oslo/PostalInformation/PostalInformationOsloController.cs [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+        }
+
+        /// <summary>
+        /// Vraag het totaal aantal van actieve postcodes op.
+        /// </summary>
+        /// <param name="context"></param>
+        /// <param name="cancellationToken"></param>
+        /// <response code="200">Als de opvraging van het totaal aantal gelukt is.</response>
+        /// <response code="500">Als er een interne fout is opgetreden.</response>
+        [HttpGet("totaal-aantal")]
```
Comment (ArneD) [inline src/PostalRegistry.Api.Oslo/PostalInformation/PostalInformationOsloController.cs:151]: Also add `[Produces(AcceptTypes.JsonLd)]`

File: src/PostalRegistry.Api.Oslo/PostalInformation/Query/PostalInformationListOsloQuery.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            //        .AsNoTracking()
+            //        .Where(x => x.NameDutchSearch == filterMunicipalityName ||
+            //                    x.NameFrenchSearch == filterMunicipalityName ||
+            //                    x.NameEnglishSearch == filterMunicipalityName ||
+            //                    x.NameGermanSearch == filterMunicipalityName)
+            //        .Select(x => x.NisCode)
+            //        .ToList();
+
+            //    postalInformationSet = postalInformationSet.Where(x => municipalityNisCodes.Contains(x.NisCode));
+            //}
```
Comment (ArneD) [inline src/PostalRegistry.Api.Oslo/PostalInformation/Query/PostalInformationListOsloQuery.cs:49]: Will have to be implemented

File: src/PostalRegistry.Api.Oslo/PostalRegistry.Api.Oslo.csproj [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    <Content Include="Dockerfile" CopyToOutputDirectory="Always" />
+    <Content Include="appsettings.json" CopyToOutputDirectory="Always" />
+    <Content Include="appsettings.*.json" CopyToOutputDirectory="Always" />
+    <Content Include="paket.template" CopyToOutputDirectory="Always" />
+    <Content Include="init.sh" CopyToOutputDirectory="Always" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\PostalRegistry.Infrastructure\PostalRegistry.Infrastructure.csproj" />
+    <ProjectReference Include="..\PostalRegistry.Projections.Legacy\PostalRegistry.Projections.Legacy.csproj" />
```
Comment (ArneD) [inline src/PostalRegistry.Api.Oslo/PostalRegistry.Api.Oslo.csproj:33]: You will need syndication reference to be able to filter on municipality name

File: src/PostalRegistry.Api.Oslo/PostalRegistry.Api.Oslo.csproj.DotSettings [hunk 4]
```diff
@@ -0,0 +1,3 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```
Comment (ArneD) [inline src/PostalRegistry.Api.Oslo/PostalRegistry.Api.Oslo.csproj.DotSettings:1]: Is this file necessary? DotSettings are configured on solution.

File: src/PostalRegistry.Api.Oslo/paket.template [hunk 5]
```diff
@@ -0,0 +1,30 @@
+type file
+version 1.0.0
+
+id Be.Vlaanderen.Basisregisters.PostalRegistry.Api.Legacy
```
Comment (ArneD) [inline src/PostalRegistry.Api.Oslo/paket.template:4]: `Be.Vlaanderen.Basisregisters.PostalRegistry.Api.Oslo` (same rule below)

File: src/PostalRegistry.Api.Oslo/paket.template [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+iconUrl https://raw.githubusercontent.com/Informatievlaanderen/build-pipeline/master/logo.png
+licenseUrl https://opensource.org/licenses/EUPL-1.2
+
+description Postal Registry.
+
+files
+  PostalRegistry.Api.Legacy.dll => lib\net5.0
+  PostalRegistry.Api.Legacy.pdb => lib\net5.0
+  PostalRegistry.Api.Legacy.xml => lib\net5.0
+  PostalRegistry.Api.Legacy.xml => content
```
Comment (ArneD) [inline src/PostalRegistry.Api.Oslo/paket.template:21]: `PostalRegistry.Api.Oslo`

**Review by ArneD — CHANGES_REQUESTED** [review]

File: src/PostalRegistry.Api.Oslo/PostalRegistry.Api.Oslo.csproj.DotSettings (hunk 4 above)
Comment (alainthoen) [reply src/PostalRegistry.Api.Oslo/PostalRegistry.Api.Oslo.csproj.DotSettings:1]: Staat ook in het Legacy project

**Review by lvermeulen — APPROVED** [review]
lgtm
